app/models/application_record.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  default_scope { where(company_id: Company.current_id) }
```

Comment:
we discussed use of `inherited` method. Discuss with basil, this isn't the right way. Urgent.

Reply:
ok done